frontend/src/components/Button.module.scss

```diff
@@ -39,10 +38,15 @@
   }
 
   &.is-destructive {
-    background-color: $color-red-50;
+    background-color: $color-red-60;
+
+    &:disabled {
+      background-color: $color-red-10;
+    }
 
     &:hover:not(:disabled) {
-      background-color: $color-red-60;
+      background-color: $color-red-70;
+      opacity: 1;
```

Comment:
Hmm, this line seems to be overwriting a style that is not defined in this module, which I think is a red flag? But I don't see any differences when I remove this line, so I'm not sure where to verify.